File: URI_1041_coordinates_of_a_point.java
import java.util.Scanner;

public class URI_1041_coordinates_of_a_point {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		double x = scanner.nextDouble();
		double y = scanner.nextDouble();

		if (x > 0.0 && y > 0.0) {
			System.out.println("Q1");
		} else if (x > 0.0 && y < 0.0) {
			System.out.println("Q4");
		} else if (x < 0.0 && y > 0.0) {
			System.out.println("Q2");
		} else if (x < 0.0 && y < 0.0) {
			System.out.println("Q3");
		} else if (x == 0.0 && y != 0.0) {
			System.out.println("Eixo Y");
		} else if (x != 0.0 && y == 0.0) {
			System.out.println("Eixo X");
		} else if (x == 0.0 && y == 0.0) {
			System.out.println("Origem");
		}

	}
}

File: URI_1041_coordinates_of_a_point.js
const lines = ["1.0 0.0"];

const inputArr = lines[0].split(" ");
const x = Number(inputArr[0]);
const y = Number(inputArr[1]);

if (x > 0 && y > 0) {
    console.log("Q1");
} else if (x > 0 && y < 0) {
    console.log("Q4");
} else if (x < 0 && y > 0) {
    console.log("Q2");
} else if (x < 0 && y < 0) {
    console.log("Q3");
} else if (x === 0 && y !== 0) {
    console.log("Eixo Y");
} else if (x !== 0 && y === 0) {
    console.log("Eixo X");
} else if (x === 0 && y === 0) {
    console.log("Origem");
}
